usmqe_tests/web/others/test_lists.py

```diff
@@ -78,7 +78,11 @@ def test_cluster_list(valid_credentials, sds_name):
         pytest.check(
             cluster.is_present,
             'Any cluster (line in Clusters list) should have some elements')
+        HostsList = cluster.get_hosts()
+        print([host.host for host in HostsList])
+        print(inventory_hosts)
         pytest.check(
-            int(cluster.hosts_nr) == len(inventory_hosts),
+            len([host for host in HostsList]) == len(inventory_hosts),
```

Comment:
please put back the original compare and add a check_hosts check
https://github.com/usmqe/usmqe-tests/blob/master/usmqe/web/tendrl/clusters/auxiliary/pages.py